src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -334,73 +332,33 @@ struct
     let root_blockchain_state =
       Protocol_state.blockchain_state root_protocol_state
     in
-    let root_blockchain_state_ledger_hash, root_blockchain_staged_ledger_hash =
-      ( Protocol_state.Blockchain_state.snarked_ledger_hash
-          root_blockchain_state
-      , Protocol_state.Blockchain_state.staged_ledger_hash
-          root_blockchain_state )
+    let root_blockchain_state_ledger_hash =
+      Protocol_state.Blockchain_state.snarked_ledger_hash root_blockchain_state
     in
     assert (
       Ledger_hash.equal
         (Ledger.Db.merkle_root root_snarked_ledger)
         (Frozen_ledger_hash.to_ledger_hash root_blockchain_state_ledger_hash)
     ) ;
-    let root_masked_ledger = Ledger.of_database root_snarked_ledger in
-    let root_snarked_ledger_hash =
-      Frozen_ledger_hash.of_ledger_hash
-      @@ Ledger.merkle_root (Ledger.of_database root_snarked_ledger)
+    let root_breadcrumb =
```

Comment:
Is setting `just_emitted_a_proof` correct now, @bkase ?

Reply:
I’m not sure if this always is false. This is true whenever root_staged_ledger just emitted a proof. Sometimes this is the one we’d bootstrap/catchup to.